operator/api/v1alpha1/common/common.go

```diff
@@ -117,13 +117,13 @@ const PreDeploymentEvaluationCheckType CheckType = "pre-eval"
 const PostDeploymentEvaluationCheckType CheckType = "post-eval"
 
 type KeptnMeters struct {
-	TaskCount          syncint64.Counter
+	TaskCount          syncint64.UpDownCounter
```

Comment:
why has this been changed to `UpDownCounter`?

Reply:
Counter has caused the problem described in the issue ticket, I didn't find a logical explanation for that, but UpDownCounter works exactly as expected and fixes the wrong data